gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -469,7 +469,9 @@ def add_arguments(self, parser):
     parser.add_argument(
         '--output_file',
         required=True,
-        help='The full path of the VCF file to store the result.')
+        help='The full path of the VCF file to store the result. If run with '
```

Comment:
nit: we have been surrounding the help text with brackets, so please add them to be consistent.